turingschool-examples/ideabox-boilerplate
# Feature/comments

### What does this PR do?
- adds comment form in pop up box
- adds comment based on user input to idea data
- filters idea cards based on their starred status
### What does it fix?
- N/A
### Is this a feature or a fix?
- feature
### Where should the reviewer start?
- all of comment.js file
- main.js lines 80-99 (functions for writing and submitting comment) and lines 149-163 (function for filtering ideas)
### How should this be tested?
- star ideas in card grid, then click "Show Starred Ideas" button on the left. Only the starred ideas should now be displayed in the card grid. Click the button again to return to the view with all ideas
- click the plus button next to "comment" on an idea card. A pop up form will appear in which you can write a comment and submit. The comment input box should close and the submitted comment should appear as an object in the corresponding locally stored idea.
